rllib/utils/policy.py

```diff
@@ -17,14 +18,48 @@
     TensorStructType,
     TensorType,
 )
+from ray.util import log_once
 from ray.util.annotations import PublicAPI
 
 if TYPE_CHECKING:
     from ray.rllib.policy.policy import Policy
 
+logger = logging.getLogger(__name__)
+
 tf1, tf, tfv = try_import_tf()
 
 
+@PublicAPI(stability="alpha")
+def validate_policy_id(policy_id: str, error: bool = False) -> None:
```

Comment:
can we please write a unit test for this util?

Reply:
Is this really necessary? This is a single line, one reg exp match function! ??